query1.php

```diff
@@ -26,32 +26,41 @@
 		<!-- grid-container class which contains all content using CSS grid -->
 		<div class="grid-container">
 		
-			<div class="nav1">
-				<b><p style = "color: white; font-size: 2vw; margin: 1vw 0vw 0vw 1vw;">Navigation</p></b>
-			</div>
-		
 			<!-- header class which consists of the header image, text and navigation system -->
 			<div class="header">
+				
 				<b><p style = "color: black; font-size: 2vw; margin: 1vw 0vw 0vw 1vw;">Header</p></b>
+				
+				<!-- navigation located at the very top -->		
+				<div class="nav" style = "top: 0vw; position: sticky;">
+					<b><p style = "color: white; font-size: 2vw; margin: 1vw 0vw 0vw 1vw;">Navigation</p></b>
+				</div>
+			
 			</div>
 			
 			<!-- search_content which contains the search system -->
-			<div class = "search_content">
-				<b><p style = "color: black; font-size: 2vw; margin: 1vw 0vw 0vw 1vw;">Search System</p></b>
+			<div class = "search_content" style="position: sticky; top: 0vw;">

```

Comment:
Added search function inside search_content. This search system will be developed and functioned in later stages.